pareto/examples/strategic_model_treatment_module_demo/strategic_model_treatment_module_demo.ipynb

```diff
@@ -0,0 +1,1662 @@
+    "    \"FreshSourcingCost\", \"TruckingHourlyCost\", \"PipelineDiameterValues\",\n",
+    "    \"DisposalCapacityIncrements\", \"InitialStorageCapacity\", \"StorageCapacityIncrements\",\n",
+    "    \"TreatmentCapacityIncrements\", \"TreatmentEfficiency\", \"RemovalEfficiency\",\n",
+    "    \"DisposalExpansionCost\", \"StorageExpansionCost\", \"TreatmentExpansionCost\",\n",
+    "    \"PipelineCapexDistanceBased\", \"PipelineCapexCapacityBased\", \"PipelineCapacityIncrements\",\n",
+    "    \"PipelineExpansionDistance\", \"Hydraulics\", \"Economics\", \"PadWaterQuality\",\n",
+    "    \"StorageInitialWaterQuality\", \"PadStorageInitialWaterQuality\", \"DisposalOperatingCapacity\",\n",
+    "]\n",
+    "\n",
+    "# Load data from Excel input file into Python\n",
+    "with resources.path(\n",
+    "    \"pareto.case_studies\",\n",
+    "    \"strategic_toy_case_study.xlsx\",\n",
+    ") as fpath:\n",
+    "    [df_sets, df_parameters] = get_data(fpath, set_list, parameter_list)\n",
+    "\n",
+    "# Create Pyomo optimization model representing the produced water network\n",
+    "strategic_model = create_model(\n",
+    "    df_sets,\n",
+    "    df_parameters,\n",
+    "    default={\n",
+    "        \"objective\": Objectives.cost,\n",
+    "        \"pipeline_cost\": PipelineCost.distance_based,\n",
+    "        \"pipeline_capacity\": PipelineCapacity.input,\n",
+    "        \"node_capacity\": True,\n",
+    "        \"water_quality\": WaterQuality.false,\n",
+    "    },\n",
+    ")\n",
+    "\n",
+    "# Solve Pyomo model with specified options\n",
+    "results_dict = solve_and_check_feasibility(strategic_model)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## <a id='sensitivity'></a>Sensitivity analysis\n",
+    "To perform sensitivity analysis, we can change model data and then re-solve the model to see the effects on the optimal solution. To demonstrate, we will change some of the costing data associated with desalination technologies to see the effect on which technology gets selected for site R01.\n",
+    "\n",
+    "First, we extract results on treatment buildout at R01 from our solved model:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Extract R01 buildout results for default solved model\n",
+    "technology, capacity = get_R01_results(results_dict)\n",
+    "print(f\"Technology: {technology}\")\n",
+    "print(f\"Capacity: {capacity}\")\n",
+    "print(f\"Objective function value: {value(strategic_model.v_Z)}\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "At site R01, the optimizer has chosen to build MVC desalination with the capacity selected corresponding to the `J1` index. As an aside, we can look at the input data to remind ourselves what capacity `J1` corresponds to (in bbl/day):"
```

Comment:
Again, the flow of instructions assumes the cells are run. So, in my opinion, it may be useful to save the notebook after cells are run. The read through will be more meaningful with references being apparent.

Reply:
See my above comment on the matter. If we want to incorporate the notebook into the documentation, it will need to be an executed version of the notebook. But for now, we are just going to commit uncommitted notebooks to the repo until we figure out exactly how we want to incorporate the notebooks into the documentation.